cdhweb/static_src/global/components/site-alert.scss

```diff
@@ -0,0 +1,114 @@
+.alert-banner {
+  --alert-accent-color: var(--color-brand-100);
+  --alert-bg-color: var(--color-brand-5);
+
+  // This linear gradient has a hard stop in the middle and goes full width.
+  // This is so we get  the left-side full bleed for the decoration area.
+  // The content area has a white background which hides this trickery.
+  background-image: linear-gradient(
+    to right,
+    var(--alert-accent-color) 50%,
+    var(--alert-bg-color) 50%
+  );
+
+  line-height: 1.5;
+  border-block-end: 1px solid var(--color-brand-110);
+}
+
+.alert-banner--informational {
+  --alert-accent-color: var(--color-brand-100);
+}
+.alert-banner--warning {
```

Comment:
You could put these selectors together with a comma to make it clear that the same happens for both.  Or set this variable on `.alert-banner` and only override it with `--emergency`